listenbrainz/webserver/views/api_tools.py

```diff
@@ -13,10 +13,12 @@
 from listenbrainz.webserver import API_LISTENED_AT_ALLOWED_SKEW
 from listenbrainz.webserver.external import messybrainz
 from werkzeug.exceptions import InternalServerError, ServiceUnavailable, BadRequest
+from listenbrainz.listenstore import RedisListenStore
```

Comment:
you should import `listenbrainz.webserver.redis_connection` and use the `redis_connection._redis` object.